vividus-plugin-json/build.gradle

```diff
@@ -0,0 +1,18 @@
+project.description = 'Vividus plugin for JSON testing'
+
+dependencies {
+    implementation project(':vividus-bdd-engine')
+    implementation project(':vividus-soft-assert')
+    implementation project(':vividus-util')
+    implementation(group: 'com.jayway.jsonpath', name: 'json-path-assert', version: '2.4.0') {
+        exclude module: 'hamcrest-library'
+    }
+    implementation (group: 'net.javacrumbs.json-unit', name: 'json-unit', version: '2.18.0')
```

Comment:
Have tried to get rid of this dependencies, but do not succeed. Will discuss tomorrow